target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -20,7 +20,6 @@ case "$FIRMWARE" in
 		;;
 	buffalo,wxr-2533dhp)
 		caldata_extract "ART" 0x1000 0x2f20
-		ath10k_patch_mac $(mtd_get_mac_binary ART 0x1e)
```

Comment:
Without `ath10k_patch_mac` we can now consolidate duplicate cases.

Reply:
Can't understand what do you mean here.

Reply:
We have lots of block with identical setup, and those can be merged. I've already done that in a separate patch.